xServerWorker/Jobs/GraviexOrderbookJob.cs

```diff
@@ -0,0 +1,155 @@
+﻿using Common.Models.Graviex;
+using Common.Models.OrderBook;
+using Common.Utils;
+using Microsoft.Extensions.Logging;
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization;
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Driver;
+using Newtonsoft.Json;
+using Quartz;
+using RestSharp;
+using System.Dynamic;
+
+namespace xServerWorker.Jobs
+{
+    [DisallowConcurrentExecution]
+    public class GraviexOrderbookJob : IJob
+    {
+        private readonly MongoClient _client;
+        private readonly IMongoDatabase _db;
+
+        public GraviexOrderbookJob()
+        {
+#if DEBUG
+            _client = new MongoClient($"mongodb://localhost:27017");
+#else
+                var mongoUser = Environment.GetEnvironmentVariable("MONGO_USER");
+                var mongoPassword = Environment.GetEnvironmentVariable("MONGO_PASSWORD");
+
+                _client = new MongoClient($"mongodb://{mongoUser}:{mongoPassword}@xDocumentStore:27017/");
```

Comment:
Perhaps it would be better to allow the user to change the whole mongodb connection string.